scriptmodules/ports/yquake2.sh

```diff
@@ -96,19 +100,23 @@ function configure_yquake2() {
 
     if isPlatform "gl3"; then
         params+=("+set vid_renderer gl3")
+    elif isPlatform "gles"; then
+        params+=("+set vid_renderer gles1")
     elif isPlatform "gl" || isPlatform "mesa"; then
         params+=("+set vid_renderer gl1")
     else
         params+=("+set vid_renderer soft")
     fi
 
-    if isPlatform "kms"; then
+    if ! isPlatform "gles" && isPlatform "kms"; then
```

Comment:
AFAICT there's no `kms` but non-`gles` system in RetroPie, so the `kms` check would be redundant. Why the new test here ? Is the new renderer not working with `r_customwidth`/`r_customheight` or you think the video mode parameters are no longer necessary ?

Reply:
My apologies, my cleanup wasn't a as complete as I thought. The mentioned verification and parameters are no longer needed, so they were taken out.
For the record, the new renderer does work with `r_customwidth` / `r_customheight` , but it's no longer necessary to put them on the parameters since the `r_mode -2` (automatic resolution) included in the new default config is more succint, and achieves the exact same result, with the possibility for the user to change it if it's needed. Vsync is also enabled by default, on all renderers.